include/tvSettings.h

```diff
@@ -2567,6 +2567,58 @@ tvError_t EnableDynamicContrast(bool mode);
  */
 tvError_t EnableLocalContrast(bool mode);
 
+/** 
+* @brief Triggers short circuit detection operation and returns the status.
+*
+* On calling GetLdimZoneShortCircuitStatus, the short circuit detection logic
+* is triggered. It detects short circuits between zones.  On completion, it
+* returns a list of zones where a short circuit was detected.
+*
+* @param[in/out] status
+*     - int*, Callee-saved status indicating short circuit detected or not.
+*     - 0 indicates success, no short circuit detected.
+*     - 1 indicates failure, at-least one short circuit detected.
+*     - This should be pre-allocated by the caller.
+*
+* @param[in/out] shortcircuit_zone_list
```

Comment:
Add as @param[out]. Dont represent as [in/out]

Reply:
will update as out param